linux_os/guide/system/software/integrity/software-integrity/rpm_verification/rpm_verify_hashes/rule.yml

```diff
@@ -11,12 +11,12 @@ description: |-
     The RPM package management system can check the hashes of
     installed software packages, including many that are important to system
     security.
-    To verify that the cryptographic hash of system files and commands match vendor
+    To verify that the cryptographic hash of system files and commands matches vendor
     values, run the following command to list which files on the system
     have hashes that differ from what is expected by the RPM database:
-    <pre>$ rpm -Va | grep '^..5'</pre>
+    <pre>$ rpm -Va --noconfig | grep '^..5'</pre>
```

Comment:
Add the Jinja2 conditional here to remove `--config` from RHEL6 rule description.
In the OCIL too.

Reply:
Done. Thanks.